models/articles.models.js

```diff
@@ -124,53 +106,54 @@ const writeComment = (article_id, { body, votes, author }) => {
   let sqlQuery = "SELECT * FROM users WHERE username = $1";
   let queryValues = [author];
 
-  return db.query(sqlQuery, queryValues).then(({ rows }) => {
-    if (!rows.length) {
-      return Promise.reject({ status: 404, msg: "User not found" });
-    } else {
-      sqlQuery = "SELECT * FROM comments WHERE article_id = $1";
-      queryValues = [article_id];
-      return db.query(sqlQuery, queryValues).then(({ rows }) => {
-        if (!rows.length) {
-          return Promise.reject({ status: 404, msg: "Article not found" });
-        } else {
-          return db
-            .query(
-              `INSERT INTO comments
-              (body, votes, article_id, author)
-              values ($1, $2, $3, $4) RETURNING *`,
-              [body, votes, article_id, author]
-            )
-            .then((result) => {
-              return result.rows[0];
-            });
-        }
-      });
-    }
-  });
+  return db
+    .query(sqlQuery, queryValues)
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "User not found" });
+      } else {
+        sqlQuery = "SELECT * FROM articles WHERE article_id = $1";
+        queryValues = [article_id];
+        return db.query(sqlQuery, queryValues);
+      }
+    })
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
+      } else {
+        return db.query(
+          `INSERT INTO comments
+          (body, votes, article_id, author)
+          values ($1, $2, $3, $4) RETURNING *`,
+          [body, votes, article_id, author]
+        );
+      }
+    })
+    .then((result) => {
+      return result.rows[0];
+    });
 };
 
 const changeArticle = (article_id, inc_votes) => {
   let sqlQuery = "SELECT votes FROM articles WHERE article_id = $1";
   let queryValues = [article_id];
 
-    return db
-      .query(sqlQuery, queryValues)
-      .then(({ rows }) => {
-        if (!rows.length) {
-          return Promise.reject({ status: 404, msg: "Article not found" });
-        } else {
-          const newVotes = inc_votes + rows[0].votes;
-          sqlQuery =
-            "UPDATE articles SET votes = $1 WHERE article_id = $2 RETURNING *";
-          queryValues = [newVotes, article_id];
-          return db.query(sqlQuery, queryValues);
-        }
-      })
-      .then(({ rows }) => {
-    
-        return rows[0];
-      });
+  return db
+    .query(sqlQuery, queryValues)
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
+      } else {
+        const newVotes = inc_votes + rows[0].votes;
+        sqlQuery =
+          "UPDATE articles SET votes = $1 WHERE article_id = $2 RETURNING *";
+        queryValues = [newVotes, article_id];
+        return db.query(sqlQuery, queryValues);
```

Comment:
instead of using a separate query, getting votes and making newVotes, you can update this directly in one update with `SET votes = votes + $1`